### ISSUE TITLE
[Feature Request]: The ability to order the subscriptions bar based on selected metrics.

### ISSUE THREAD

**Issue by SentientBeingOnAQuest** [FreeTubeApp/FreeTube#4096]
### Guidelines

- [X] I have searched the issue tracker for [open](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aopen+is%3Aissue) and [closed](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aissue+is%3Aclosed) issues that are similar to the feature request I want to file, without success.
- [X] I have searched the [documentation](https://docs.freetubeapp.io/) for information that matches the description of the feature request I want to file, without success.
- [X] This issue contains only one feature request.

### Problem Description

As it currently stands the subscription list appears random and chaotic. 
I would like to see an options menu that allows users to order and group their subscriptions in the sidebar based on metrics such as: alphabetical, subscription date, subscription profile, subscriber count ect. 

### Proposed Solution

Add an options menu that allows users to order and group their subscriptions in the sidebar based on metrics such as: alphabetical, subscription date, subscription profile, subscriber count ect. 

### Alternatives Considered

Allow users to customize the order of the subscriptions shown in the sidebar with a drag and drop slider. This would be inferior to the proposed solution, but would provide a solution nonetheless. 

### Issue Labels

ease of use improvement, improvement to existing feature

**Comment by absidue** [comment]
The subscriptions list in the sidebar is already ordered alphabetically?

**Comment by SentientBeingOnAQuest** [comment]
> The subscriptions list in the sidebar is already ordered alphabetically?

True, but it would have to be added as an option if other options for ordering would be given.
